.github/workflows/ci.yaml

```diff
@@ -43,4 +43,4 @@ jobs:
         run: make lint skip-non-generated-files=true
       - name: Test
         working-directory: ./sdk-repo-updated
-        run: make test skip-non-generated-files=true
\ No newline at end of file
+        run: make test skip-non-generated-files=true
```

Comment:
what changed here? :thinking:

Reply:
I think nothing, this sometimes happens in the last line of the file I don't know why